students/HarveyHerela/lesson05/mailroom3.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+
+import datetime
+import os
+
+donors = [{"name": ("Chase", "Zuma"), "donations": [25 * x for x in range(2)]},
+          {"name": ("Marshall", "Rocky"), "donations": [30 * x for x in range(2, 4)]},
+          {"name": ("Skye", "Rubble"), "donations": [45 * x for x in range(10, 13)]}]
```

Comment:
Don't think I'd use comprehensions here, since it's more readable to see the donation amounts listed out one at a time